obj/baum.h

```diff
@@ -26,7 +29,7 @@ class baum_t : public obj_t
 	static PLAYER_COLOR_VAL outline_color;
 
 	/** month of birth */
-	uint16 geburt;
+	uint16 purchase_time;
```

Comment:
The new name of the variable is misleading - trees are spawned, not purchased by a player.
```suggestion
	uint16 spawn_time;
```

Reply:
The name is for consistency. City buildings also have a 'purchase_time' which is used as the time of original building (I suppose this is called "spawn" in game jargon) although it is quite rare for players to purchase buildings.  Clouds (wolke.cc) also have a purchase_time although players cannot purchase clouds. Purchase_time as a name presumably came from vehicles, where it makes sense.  Perhaps "creation_time" would be more sensible as a generic name, but can we keep purchase_time in the spirit of generic object-orientedness?